src/Compilers/Core/Portable/Emit/NoPia/CommonEmbeddedMember.cs

```diff
@@ -45,14 +45,14 @@ protected CommonEmbeddedMember(TMember underlyingSymbol)
                 this.UnderlyingSymbol = underlyingSymbol;
             }
 
-            protected abstract IEnumerable<TAttributeData> GetCustomAttributesToEmit(TModuleCompilationState compilationState);
+            protected abstract IEnumerable<TAttributeData> GetCustomAttributesToEmit(TPEModuleBuilder moduleBuilder);
```

Comment:
This is needed to pass to underlying symbols, as their APIs changed.

---
In reply to: [115340980](https://github.com/dotnet/roslyn/pull/18715#discussion_r115340980) [](ancestors = 115340980)